packages/rollup-plugin/src/index.ts

```diff
@@ -135,9 +135,12 @@ export default function typegpu(): Plugin {
             ) {
               const implementation = node.arguments[0];
 
-              if (implementation) {
+              if (
+                implementation &&
+                !(implementation.type === 'TemplateLiteral')
```

Comment:
por que?

Reply:
To not run the plugin when `does` is a string implementation of the function

Reply:
will probably fail with one-line string implementations ("..." or '...')